package.json

```diff
@@ -5,9 +5,11 @@
   "eggPlugin": {
     "name": "redis"
   },
+  "types": "./index.d.ts",
```

Comment:
Also need to add `@types/ioredis` in dependencies?

Reply:
`ioredis` created `index.d.ts` for use it directly.

Reply:
Are you sure ? https://github.com/luin/ioredis But I can't find any types file in this repo